examples/deployments/EKS/README.md

```
@@ -0,0 +1,269 @@
+# Steps to create an EKS cluster
+
+This guide demonstrates Dynamo platform on Amazon Elastic Kuberentes Service (EKS) platform.
+
+## Install CLIs
+
+### Install AWS CLI (steps [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html))
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)